pkg/minikube/problem/err_map.go

```diff
@@ -61,6 +61,11 @@ var vmProblems = map[string]match{
 		Advice: "Disable Hyper-V when you want to run VirtualBox to boot the VM",
 		Issues: []int{4051},
 	},
+	"HYPERV_NO_VSWITCH": {
+		Regexp: re(`no External vswitch found. A valid vswitch must be available for this command to run.`),
```

Comment:
Please add a test.

Reply:
Do we have tests for this file anywhere?

Reply:
well.. I guess not! I swore I wrote some... hum.